src/RequestProcessor.cs

```diff
@@ -254,6 +283,37 @@ internal StreamingMessage ProcessFunctionEnvironmentReloadRequest(StreamingMessa
 
         #region Helper_Methods
 
+        /// <summary>
+        /// Initialize the worker based on the FunctionApp that the worker will deal with.
+        /// </summary>
+        private void InitializeForFunctionApp(StreamingMessage request, StreamingMessage response)
+        {
+            var functionLoadRequest = request.FunctionLoadRequest;
+
+            // If 'ManagedDependencyEnabled' is true and !DependencyManager.FunctionAppDependenciesInstalled,
+            // process the function app dependencies as defined in FunctionAppRoot\requirements.psd1.
+            // These dependencies will be installed via 'Save-Module' when the first PowerShellManager instance is being created.
+            if (functionLoadRequest.ManagedDependencyEnabled && !DependencyManager.FunctionAppDependenciesInstalled)
```

Comment:
No need for `!DependencyManager.FunctionAppDependenciesInstalled` part (here and below), as this method is called only once.

Reply:
Fixed.